From 88948b34a872ed61a16fff488040e88c6d535248 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 27 Apr 2023 20:39:00 +0100
Subject: [PATCH 13/14] gnu: python-pytest-mpl: Update to 0.16.1.

* gnu/packages/python-check.scm (python-pytest-mpl): Update to 0.16.1.
[build-system]: Use pyproject-build-system.
[arguments]<phases>: Simplify phases by removing redundant modification
and use <test-flags> instead.
[propagated-inputs]: Add python-jinja2, python-packaging.
---
 gnu/packages/python-check.scm | 19 ++++++++++---------
 1 file changed, 10 insertions(+), 9 deletions(-)

diff --git a/gnu/packages/python-check.scm b/gnu/packages/python-check.scm
index 9a61f59d56..d939ade03b 100644
--- a/gnu/packages/python-check.scm
+++ b/gnu/packages/python-check.scm
@@ -831,24 +831,25 @@ (define-public python-pytest-mockito
 (define-public python-pytest-mpl
   (package
     (name "python-pytest-mpl")
-    (version "0.11")
+    (version "0.16.1")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "pytest-mpl" version))
        (sha256
-        (base32 "1km202c1s5kcn52fx0266p06qb34va3warcby594dh6vixxa9i96"))))
-    (build-system python-build-system)
+        (base32 "0sa4229xkkah3fdg9wnqnvb9j13xsd3x1h5rwbsgb3sf2a0icmrd"))))
+    (build-system pyproject-build-system)
     (arguments
-     '(#:phases
-       (modify-phases %standard-phases
-         (replace 'check
-           (lambda _
-             (invoke "pytest" "-vv"))))))
+     (list
+      #:test-flags #~(list "-m" "mpl_image_compare")))
     (native-inputs
      (list python-pytest))
     (propagated-inputs
-     (list python-matplotlib python-nose python-pillow))
+     (list python-jinja2
+           python-matplotlib
+           python-nose
+           python-packaging
+           python-pillow))
     (home-page "https://github.com/matplotlib/pytest-mpl")
     (synopsis "Pytest plugin to help with testing figures output from Matplotlib")
     (description
-- 
2.39.2

